### ISSUE TITLE
gopro fusion gives out empty files

### ISSUE THREAD

**Issue by goamberg** [stilldavid/gopro-utils#15]
Hi 
I tested it on the gopro fusion and it gives out empty cvs files.
If you want I could send you short vids for testing?

Cheers
G

ps. thanks for the app. used it already on gopro 5!

**Comment by stilldavid** [comment]
Yeah, I'd love a sample file from the Fusion! Do you have someplace to upload it like dropbox/google drive/etc..? Thanks!

**Comment by goamberg** [comment]
Sent you an email with a link.
